keys.go

```diff
@@ -92,19 +95,26 @@ type KeyVersion struct {
 }
 
 // CreateKey creates a new KP key.
-func (c *Client) CreateKey(ctx context.Context, name string, expiration *time.Time, extractable bool) (*Key, error) {
-	return c.CreateImportedKey(ctx, name, expiration, "", "", "", extractable)
+func (c *Client) CreateKey(ctx context.Context, name string, expiration *time.Time, extractable bool, otherParameters map[string]interface{}) (*Key, error) {
```

Comment:
Lets not change the "Create*Key" methods, since those will all require updates and need to remain compatible with older versions of the API here.

```suggestion
func (c *Client) CreateKey(ctx context.Context, name string, expiration *time.Time, extractable bool) (*Key, error) {
```